docs/serverless-v2/06-01-function-cr.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Function
+type: Custom Resource
+---
+
+The `functions.serverless.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to manage lambdas within Kyma. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
```

Comment:
```suggestion
The `functions.serverless.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to manage lambdas within Kyma. To get the up-to-date CRD and show the output in the YAML format, run this command:
```
AFAIK, we agreed on YAML when talking about format, and `.yaml` when writing about a file extension. However, this part is taken from the template, so 🤔